packages/slice-machine/server/src/api/services/tracker.ts

```diff
@@ -42,6 +46,7 @@ export class ServerTracker {
   }
 
   private groupEvent(traits: Record<string, unknown>): void {
+    if (this.tracking === false) return void 0;
```

Comment:
return alone is sufficient no ?

Reply:
Yes, but I was also dealing with extra linter warnings.
I think I've found something better :)